### ISSUE TITLE
Deer need a stronger will to live

### ISSUE THREAD

**Issue by FewerFlaws** [FaeCraft/GentleFawn#5]
1. Deer let you approach.  They should use the ocelot behaviour to run away unprovoked.
2. Deer flee more slowly than cows.  They should get a speed boost when panicking due to an attack.

**Comment by ZaneDragonBorn** [comment]
Yeah, They need more health and should run when you're near them. Maybe they don't run if you crouch towards them without a weapon. But run if you equip your weapon or if you run at them.
